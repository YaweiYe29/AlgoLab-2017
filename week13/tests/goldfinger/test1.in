17
75 5 0
849813
-5 -12 3
188 -191 7
-256 218 4
-33 -184 3
-46 -3 5
-239 -252 7
196 -156 5
-202 234 4
161 151 5
80 33 6
181 28 7
-223 -133 3
100 -111 4
-11 -241 8
251 151 2
244 89 3
90 -178 4
45 -59 4
-55 -243 4
81 -171 3
84 -154 6
54 -194 3
-64 238 5
0 -108 7
-225 218 3
-233 211 2
27 175 4
221 45 8
95 179 5
180 188 4
-124 -249 6
-173 1 2
219 166 3
157 51 5
171 -30 2
-68 106 5
-103 -46 7
-186 186 3
23 146 2
57 146 5
-134 58 5
-229 35 3
-113 182 6
69 29 4
-47 150 6
224 -163 5
-192 -127 6
-254 218 3
-109 -106 7
52 -200 3
-142 97 6
200 -249 3
-125 -238 8
-53 -69 7
208 248 7
112 31 7
-77 -153 3
202 210 6
206 182 8
-104 -163 6
-55 -218 7
216 -209 5
-71 95 8
97 225 5
-45 156 2
-75 214 5
-86 32 3
142 -10 5
36 -51 5
-172 -99 3
57 134 6
-208 -187 3
90 -47 7
-128 -152 7
106 -247 5
209 216
-188 -30
-93 237
-97 222
205 16
64 17 0
993199
-231 -55 7
222 -21 3
-193 75 4
-157 252 6
55 134 8
-84 71 2
-205 -108 5
-54 -136 4
240 125 7
-222 -29 8
-15 226 5
-195 -239 3
-15 -196 2
85 -242 7
18 139 7
212 177 5
74 58 3
-163 -238 6
-38 123 4
-114 185 2
-2 -201 4
-5 -240 7
228 -120 4
-92 -65 2
-112 -34 8
-215 175 8
54 -112 2
233 -210 4
5 152 7
183 8 4
112 -157 3
-132 -81 8
57 -100 8
-255 -37 5
85 214 7
44 -25 6
217 -134 8
231 139 8
174 -79 4
-146 238 7
131 170 8
209 164 2
-39 108 8
85 168 5
84 256 3
-110 -87 6
119 249 7
121 -6 8
22 137 7
180 151 3
158 -188 2
-32 11 2
177 -45 8
28 29 5
234 -231 7
-63 27 5
-13 -8 4
-23 6 8
-157 -132 5
-124 35 2
-225 81 8
-117 118 4
-107 -50 5
-37 -227 3
-200 30
241 -49
137 49
-5 -97
97 -172
140 -117
157 116
-179 46
247 179
163 108
115 -211
-42 -43
-229 54
-1 100
35 155
-143 -218
-226 -245
36 10 0
953851
-101 -42 7
140 131 5
222 184 5
1 -13 5
-186 31 8
129 176 6
-193 -221 2
-199 110 7
244 37 2
97 -108 8
84 121 3
-4 37 2
248 -137 3
127 128 7
76 142 3
-102 52 6
69 -113 8
-67 98 2
40 -248 4
84 -185 2
-130 31 2
-86 18 4
-173 -19 4
46 -131 8
45 -194 4
180 -61 7
-96 -38 2
55 95 5
-42 -27 8
87 -113 5
-226 -28 2
200 -10 7
25 -170 5
-78 -10 4
143 85 3
-91 170 7
-17 9
-3 209
152 231
-83 193
-210 -55
-249 20
13 -118
-89 108
29 126
76 55
16 19 0
974185
-107 193 7
-203 -133 7
148 218 8
132 -187 7
-133 -2 6
124 28 7
-121 -171 6
159 252 7
192 -3 5
173 170 5
204 15 4
74 168 2
141 5 8
90 154 6
105 238 2
-79 54 4
143 84
-129 113
251 -171
-71 226
140 37
0 -246
48 -70
52 -167
247 97
-203 79
181 64
195 75
-226 65
-165 225
-48 14
120 69
-248 -155
-106 -23
-238 -13
19 5 0
167847
112 -193 7
228 -33 2
234 84 5
-182 207 5
-153 -209 2
183 121 3
249 -74 7
-151 49 3
-141 -10 4
-154 -241 6
204 129 8
196 73 8
-187 -96 7
-40 59 2
241 69 2
-179 -106 7
124 -41 6
-14 157 5
29 163 2
-173 166
-89 241
-1 -29
-51 143
-54 134
44 8 0
736427
109 -8 8
5 145 7
-7 -35 5
-120 -198 2
34 -117 5
-240 208 7
-186 240 6
152 -161 7
92 -219 2
-31 -36 5
52 86 5
158 -36 7
-256 -140 4
-256 38 5
-221 249 7
-26 42 4
-246 160 7
-202 -199 8
-194 -194 3
170 35 3
-222 -104 8
163 209 7
-44 223 5
246 189 2
68 39 2
42 -214 8
235 190 3
4 40 6
-46 -30 3
-96 -99 8
71 -158 7
173 -244 5
-131 133 2
208 35 6
-197 -213 6
165 -219 3
26 237 7
230 -93 4
-90 -24 4
187 199 3
180 89 4
190 -169 3
161 42 6
188 -227 7
186 -193
-64 149
0 181
-82 189
0 -38
216 125
-66 210
81 212
15 15 0
327544
-60 211 5
-57 -144 3
-74 -185 3
-3 -37 4
255 160 4
38 -117 7
14 28 7
142 -20 2
52 -194 6
119 -169 5
-55 -16 3
201 -255 5
64 131 8
-185 237 4
-240 106 3
-179 -209
-242 -256
77 -153
253 -190
248 -65
-109 210
213 140
-125 -205
104 -17
-198 -144
47 237
206 -167
120 17
-152 -155
227 -59
52 20 0
478409
-147 208 7
93 222 6
-159 193 4
37 -138 2
-10 -95 3
-228 -169 8
-69 51 8
-133 178 3
-176 56 4
-126 135 3
124 215 5
17 -32 5
-11 -150 7
-92 -159 7
128 -28 5
-28 136 3
235 -106 8
-16 -141 6
-37 -49 2
-242 170 7
-237 82 2
109 53 6
68 -186 6
-74 69 7
-109 178 5
-230 201 4
54 50 8
237 -166 2
-72 26 6
40 -38 2
-17 -56 5
-150 -25 4
96 -20 2
50 145 2
217 -73 3
45 -205 8
-161 -175 4
206 -256 3
-7 64 8
-39 -72 6
-100 -58 4
247 -157 5
-127 245 8
-135 38 4
1 106 4
-255 -159 6
23 131 5
-218 194 7
-30 -7 7
-80 192 2
208 -222 8
42 -23 6
-126 -13
81 96
254 73
0 115
-80 -153
181 -178
-95 189
143 -166
-19 -167
80 124
-41 -111
-219 -9
-243 -141
227 174
-171 -167
55 66
-238 -171
-2 13
-141 65
-199 234
92 19 0
785125
-170 -75 5
-3 89 2
72 78 8
77 133 6
135 -154 5
-236 199 6
147 -108 7
-189 -77 5
-69 203 8
227 -223 7
-11 -53 3
54 170 6
-30 -230 2
163 233 8
-121 -138 8
75 -231 8
213 -153 3
-132 -95 2
125 -15 6
0 -122 8
-208 -81 4
108 -158 8
249 -222 4
165 -219 6
256 -78 3
251 55 2
-185 168 2
-89 -7 5
-109 106 6
-164 -154 5
159 28 3
-204 151 7
231 8 2
27 113 6
241 244 4
229 -127 4
-77 242 7
-206 -174 4
17 121 8
-246 2 7
-173 185 5
184 -109 4
44 -57 7
-39 113 3
60 -175 4
19 -131 7
-56 -69 2
60 145 5
253 142 3
-138 99 7
80 41 7
-220 243 8
12 -229 7
46 -53 8
-162 -70 3
251 -26 5
-200 -106 8
58 -173 2
-44 250 3
-75 144 2
226 -206 6
-141 239 7
88 81 8
54 29 6
-32 162 2
-204 -145 7
111 -85 3
-221 -192 5
-74 244 2
-170 -49 5
31 -6 8
99 9 5
-205 131 3
-125 -71 6
-115 4 6
166 65 5
151 -22 3
247 -36 7
94 -125 6
-194 -36 7
230 -183 2
-224 -198 6
182 -142 2
99 153 4
-197 0 3
45 216 6
191 -154 5
6 12 5
-30 156 6
-132 239 4
49 -161 8
-110 -128 4
-41 87
-212 171
163 -86
-178 142
-140 141
209 -8
-68 13
-96 75
125 181
-239 110
-248 -129
-8 -33
-24 8
-3 86
-10 154
76 -82
-58 -168
-68 -181
-198 -211
50 9 0
527635
-106 212 3
-243 -87 4
160 -88 7
119 -94 6
185 21 7
38 170 3
-141 209 6
157 112 4
-90 -71 6
-99 -6 6
-97 58 2
158 -162 3
165 -141 5
100 -118 4
253 -123 3
-23 135 2
-143 -5 8
44 243 6
-156 -5 8
92 187 7
-244 -87 3
110 132 3
-245 193 8
129 50 2
104 26 3
-253 23 2
30 -19 7
-118 -36 3
245 179 7
-180 -74 6
252 -237 5
2 171 4
119 233 3
246 -208 6
-92 -227 4
-160 -67 7
-65 61 4
231 92 4
-238 80 2
199 44 4
-144 -20 6
-102 -49 3
109 71 8
75 54 6
-67 99 3
12 127 6
-132 -139 2
-45 -143 7
-179 -26 3
-181 -63 3
-98 -28
-21 214
155 -175
-65 -145
-44 25
174 113
238 173
188 225
-108 231
52 6 0
894061
-198 -250 5
93 219 5
-108 31 3
-55 128 8
-256 -122 4
27 -204 7
191 121 7
-234 -100 7
67 47 7
-238 -176 7
-243 -99 5
-101 219 5
92 -21 3
-222 -160 3
-22 252 3
-132 64 5
-214 248 6
-149 49 2
-56 -177 8
85 105 2
-230 134 5
-54 238 7
-243 96 6
-5 -3 3
213 250 7
204 8 5
-142 -245 7
-47 25 4
221 8 2
212 151 6
187 -248 2
-18 -109 6
-223 32 3
110 153 8
-186 37 4
-118 234 6
-255 225 7
-15 -215 5
32 84 5
-223 19 3
49 144 8
-99 51 3
-84 174 2
-209 -199 7
240 152 8
72 58 3
150 -164 6
7 197 7
-26 58 8
61 -75 5
-143 188 4
-237 -209 7
195 85
178 -147
-19 215
121 -9
-79 -117
74 55
51 20 0
822696
-145 -50 6
-167 226 3
-28 -153 6
-70 102 2
-115 250 4
174 25 3
-80 122 8
233 -51 5
0 200 8
27 -59 5
-162 133 8
150 213 4
109 -180 6
67 181 3
-40 181 4
-248 -59 7
239 176 2
-9 -12 7
-153 122 8
-128 -118 7
216 -47 2
-8 -119 5
-64 93 3
-229 105 3
-203 -188 4
-200 -10 4
-46 137 6
-242 192 4
39 -78 5
-142 194 6
157 -46 4
230 186 7
-130 -171 6
-209 256 7
7 147 7
-194 50 2
238 194 8
-4 197 5
-44 98 3
-232 -37 8
-247 -119 5
-191 196 8
-207 -130 8
19 230 6
14 174 4
-143 -18 5
1 103 6
130 68 2
86 64 3
95 215 8
151 -148 7
-22 -177
-225 -128
122 -171
217 -81
26 102
-1 49
-60 -152
-27 -121
-93 -212
217 136
-244 4
-131 3
91 118
-60 -47
248 102
229 -171
181 -235
134 -212
175 38
7 -17
17 15 0
623446
-78 -251 2
-85 -250 7
-92 126 6
187 177 3
46 -167 3
-39 168 6
163 76 5
173 204 4
-97 148 5
-143 110 2
-236 -36 5
-139 242 5
59 -16 5
126 52 5
-14 -35 8
187 -104 2
68 -170 3
-223 -184
90 -215
-76 -20
-252 35
-2 -239
-124 -182
253 63
-176 -251
-218 139
180 -90
-19 223
25 188
-249 -48
201 -40
-50 72
15 10 0
863585
-250 -113 7
144 141 4
254 -115 5
28 -172 6
-229 -44 7
-45 206 8
132 -147 3
51 -122 5
-59 250 3
116 -26 5
-130 33 4
-80 67 4
108 -53 4
-83 -43 3
-20 -54 2
158 -45
62 -227
-102 191
91 5
72 -24
203 193
-95 118
-26 -82
177 110
0 39
38 5 0
977387
-251 -94 3
204 60 2
-71 198 2
8 63 7
135 -221 6
212 -195 3
189 194 3
-242 23 6
-64 -138 2
239 200 7
-123 -83 3
39 48 4
117 -158 3
-246 -175 4
233 -72 3
-27 -25 5
177 -253 6
194 113 6
55 191 6
208 -83 6
239 -18 6
-68 167 6
182 230 7
3 213 6
-42 91 3
111 174 6
143 180 8
196 -17 4
-43 -61 3
-250 -66 3
109 142 7
-208 -176 7
-113 -130 8
11 -184 4
129 -160 8
84 -42 8
-63 223 5
-249 199 6
-94 -241
-63 -249
-28 74
210 -132
-167 -181
86 7 0
469866
217 105 2
144 130 2
228 248 2
-76 82 2
-107 -164 3
-93 -201 4
241 -126 4
-26 -19 7
107 -157 7
-75 57 5
238 163 3
-235 -142 6
67 215 4
81 148 7
-237 -26 7
145 196 3
39 102 2
-166 77 6
-8 -241 2
-88 -88 6
176 -36 7
26 -247 6
53 56 3
-112 0 5
-249 -144 8
-138 88 4
-44 98 8
247 -58 2
100 207 5
-58 196 4
-86 86 3
-152 -159 4
172 115 2
-247 -124 5
43 -52 8
-177 128 7
-31 -89 3
-165 198 7
121 -168 8
-100 208 5
97 172 3
219 186 6
-216 -10 3
62 216 6
-164 173 7
205 -13 8
200 -7 2
-165 196 2
-152 -67 2
-238 -86 5
244 -58 6
-198 108 5
127 -51 3
-148 109 2
21 -84 2
170 -61 4
17 -91 2
-4 116 2
-161 -248 5
59 -82 2
197 -216 8
-63 231 4
-27 61 8
191 -64 5
-123 120 3
-3 216 4
-107 -106 8
-135 -71 6
192 -71 3
227 -91 8
-166 125 5
3 154 8
-217 35 7
224 18 4
-197 28 7
-24 -4 4
-158 -27 7
-230 -231 4
51 -25 2
-219 -232 5
28 -167 8
115 130 6
-71 -179 3
97 -204 6
204 224 5
143 -235 2
-2 35
225 150
-98 -208
-16 157
-81 -202
-159 -72
95 103
33 13 0
994247
120 156 7
-53 226 3
-5 -144 2
86 176 2
77 -2 7
228 31 7
74 242 6
-144 137 8
55 -71 2
32 -177 2
-143 110 6
-190 -216 4
-24 241 8
211 -157 5
211 25 3
143 -185 7
-255 -112 3
125 -225 8
45 -123 3
245 83 2
-139 219 6
-109 -64 8
-5 -106 4
231 194 8
106 206 2
-210 -45 6
-37 91 3
243 205 4
-22 14 8
-199 -145 6
-93 -8 2
-218 146 7
-177 -154 8
26 226
-40 -150
138 232
172 5
5 15
190 -97
183 -105
-213 -161
139 -142
-148 85
230 84
245 200
45 105
