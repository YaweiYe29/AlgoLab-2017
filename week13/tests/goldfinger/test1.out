5
2
1
2
impossible
5
7
6
3
6
5
5
3
3
3
impossible
2
